tests/syntax-tests/create_highlighted_versions.py

```diff
@@ -31,16 +42,23 @@ def create_highlighted_versions(output_basepath):
             env.pop("BAT_TABS", None)
             env["COLORTERM"] = "truecolor"  # make sure to output 24bit colors
 
-            bat_output = subprocess.check_output(
-                ["bat"] + BAT_OPTIONS + [source], stderr=subprocess.PIPE, env=env,
-            )
-
-            source_dirname = path.basename(path.dirname(source))
+            source_dirpath = path.dirname(source)
+            source_dirname = path.basename(source_dirpath)
```

Comment:
maybe add
``` python
bat_options_path = path.join(source_dirpath, "bat_options")
```
here and then check for existence + pass it in to `get_extra_options` to avoid the duplication?

Reply:
Another idea would be to return an empty list from `get_extra_options` in case the file is not there (move the existence check to the function). This way, we can just use
``` python
["bat"] + get_extra_options(source) + [source]
```